dapps/marketplace/src/components/Reviews.js

```diff
@@ -62,6 +62,10 @@ export default class Reviews extends Component {
     const { id, after } = this.props
     const first = this.props.first || 3
 
+    if (!id) {
+      console.error('Error: Reviews: id missing!')
+    }
+
     return (
       <Query
         query={query}
```

Comment:
lets try adding `skip={!id}` to the `Query` component and remove the console.error above. The query shouldn't be getting run at all if no id is set